crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -476,6 +476,9 @@ en:
       nodes_not_ready: | 
         Some nodes are not ready: %{nodes}.
         Fix the state of nodes before proceeding with the upgrade.
+      admin_missing_dns_server: |
+        The admin node does not have the dns-server role applied. During the upgrade process it is 
+        required that the admin node provides this service. Please update your deployment before continuing.
```

Comment:
I'd rephrase to:

"The Administration Server does not have the dns-server role while this is mandatory for the upgrade process. Reapply the DNS proposal with this change before proceeding with the upgrade."